### ISSUE TITLE
Anglo-Saxon Structure Specific Names

### ISSUE THREAD

**Issue by Andrettin** [0ADMods/millenniumad#28]
At the moment only the Anglo-Saxon Fortress and City Wall have specific names. Since I have done research on Old English, I would like to suggest specific names for some of the structures; for a couple of structures I have provided multiple possible alternatives. Sources for the Old English words are included.

Blacksmith: "Smiþþe" (means "smithy"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 264.
Farmstead: "Þorp" (means both "manor" and "village"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 100.
House: "Ærn" (means "house"). Source: Sean Crist, "An Analysis of *z loss in West Germanic", 2002, p. 9.
Temple: "Ealh" (means "temple"). Source: Frank Stenton, "Anglo-Saxon England", 1971, p. 101.
Temple (alternative): "Cirice" (means "church"). Source: D. H. Green, "From Germania to Europe: the Evidence of Language and History", 1997, p. xxxvi.
Temple (alternative): "Mynster" (means "monastery"). Source: Frank Stenton, "Anglo-Saxon England", 1971, p. 148.
Temple (alternative): "Weoh" (means "shrine", "idol" and "sacred precinct"). Source: Frank Stenton, "Anglo-Saxon England", 1971, p. 101.

Some units are also lacking specific names. As such I suggest:
Healer: "Munuc" (means "monk"). Source: Alaric Hall, "The Meanings of Elf and Elves in Medieval England", 2004, p. 89.
Woman: "Cwene" (means "woman"). Source: Winfred P. Lehmann, "A Grammar of Proto-Germanic", 2005, Chapter 2, Section 5.
Woman (alternative): "Wif" (means "woman"). Source: Alaric Hall, "The Meanings of Elf and Elves in Medieval England", 2004, p. 85.
Woman (alternative): "Wifmann" (means "woman-person"). Source: Alaric Hall, "The Meanings of Elf and Elves in Medieval England", 2004, p. 197.

I hope this helps :)

**Comment by StanleySweet** [comment]
Hey 
Thanks for the research !!
Since you provided reference I'm inclined to trust you on this. If you know how to you can make a pull request and I'll merge it :)
If not I'll make sure we use those for the next release :)

**Comment by Andrettin** [comment]
Hey Stan!

Sure, I can make a pull request :) I've put it up now.

I've done some more research, and found a few more:
Barracks: "Set" (means "army camp", and also "barn", "pen" and "sunset"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 214.
Corral: "Falod" (means both "sheepfold" and "ox barn"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 123.
Corral (alternative): "Stig" (means "pigsty"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 245.
Corral (alternative): "Bós" (means "cowshed"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 135.
Corral (alternative): "Penn" (means "pen"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 115.
Dock: "Hæfen" (means "harbor"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 45.
Farmstead: "Tūn" (means "farm", but also "fence around the house", "garden", "village" and "town"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 89.
Field: "Feld" (means both "field" and "area"). Source: August Fick and Alf Torp, "Wortschatz der Germanischen Spracheinheit", 2006, p. 124.

For the corral, I think "falod" is the best choice, since it can mean a corral for both sheep and oxes, and thus is more inclusive than "stig" or "bós" which mean exclusively corrals for pigs and cows respectively. And a barn is, I think, closer to what the Corral building should represent than a pen.

For the farmstead, I think "tūn" fits better than "þorp", since it actually means a farm instead of a manor.

For the temple, I chose "ealh", since it fits for both pre-Christian and Christian times. The "munuc" specific name for the healer I ended up leaving out, since it only fits for Christian times, and I imagine we would want the civilization to represent the Anglo-Saxons both before and after Christianization in its specific names.
